cmd/podman/machine/list.go

```diff
@@ -57,15 +62,29 @@ func init() {
 	_ = lsCmd.RegisterFlagCompletionFunc(formatFlagName, common.AutocompleteFormat(&entities.ListReporter{}))
 	flags.BoolVarP(&listFlag.noHeading, "noheading", "n", false, "Do not print headers")
 	flags.BoolVarP(&listFlag.quiet, "quiet", "q", false, "Show only machine names")
+	flags.BoolVar(&listFlag.allProviders, "all-providers", false, "Show machines from all providers")
```

Comment:
definitely not a blocker and it's something that could always be done later, but a `-a` option might be nice to have

Reply:
I think historically -a is short for --all, and I think --all-providers is a bit different, so I'll leave it as-is for now